lib/brainstem/api_docs/formatters/markdown/endpoint_formatter.rb

```diff
@@ -163,12 +164,29 @@ def format_presents!
               end
             end
           end
+
+          def format_custom_response_message(response_config)
+            result = "The resulting JSON is "
```

Comment:
Since we're supporting `produces`, is hard-coding JSON still desired in the documentation?

Reply:
Good call. I'll fix it

Reply:
I looked into it some more. This method will be triggered only if you use the custom response DSL which supports response as hashes or arrays. Currently produces is not used to generate any documentation about responses, developers always override the method to add their own text